SQRLDotNetClientUI/Platform/Linux/NotifyIcon.cs

```diff
@@ -79,12 +81,4 @@ private void UpdateMenu()
           
 
         }
-
-        public NotifyIcon()
-        {
-          
-           
-            
-        }
-    }
```

Comment:
Oops :)

Reply:
Sigh sorry, lol
 too fast with the clicky... so yes (haha)  fast developer you say.... but (as you can see) not as careful sometimes as I should be HAHA!
This is fixed will merge when tests are done.

Reply:
No worries, Jose, I'm glad to "clean up" behind you. 😁 
(I'm sure you can already tell that I'm a bit of a stickler when it comes to writing code, sorry for that).

Reply:
yes, but I really appreciate it! Keeps me in line I can get sloppy when I'm trying to do too many things at once. So thank you! and keep it up, doesn't bother me one bit.